pkg/endpoint/policy.go

```diff
@@ -127,17 +128,28 @@ func (e *Endpoint) removeOldFilter(owner Owner, labelsMap *LabelsMap,
 		for _, id := range getSecurityIdentities(labelsMap, &sel) {
 			srcID := id.Uint32()
 			ruleCtx := policy.RuleContext{
-				SecID: id,
-				Port:  port,
-				Proto: proto,
+				SecID:          id,
+				Port:           byteorder.HostToNetwork(port).(uint16),
```

Comment:
I wonder if we could get the compiler to type-check the byteorder...

Reply:
I don't think we can